.github/workflows/tests.yml

```diff
@@ -101,6 +103,7 @@ jobs:
         run: .github/scripts/build.sh
         env:
           OPENSSL_ROOT_DIR: /usr/local/opt/openssl/
+          ARGS:
```

Comment:
Y no ninja in macos? 😭

Reply:
it's failing with ninja on macos, we did not understand why